test/test_checks.py

```diff
@@ -1,6 +1,18 @@
 from flake8_quotes import QuoteChecker
 import os
-from unittest import TestCase
+import subprocess
+from unittest import expectedFailure, TestCase
+
+
+FLAKE8_VERSION = int(os.environ.get('FLAKE8_VERSION', 2))
```

Comment:
Ah, sorry. I must have forgotten to add quotes to the fallback =/ We aren't using the actual semver but it's the same category of environment variables/version numbers (which are strings)